src/index.ts

```diff
@@ -1,13 +1,17 @@
 export { default as BXButton } from './components/button/button';
-export { default as BXTable } from './components/data-table/table';
```

Comment:
Was the removals in this file intentional, or did something happen with rebasing...?

Reply:
OK do you want to fix this? I can do manual merge here if you'd like - Let me know. Thanks!

Reply:
I don't _think_ there's anything to fix? Let me know if there's an export that's missing though!

Reply:
Uh - I now see what you meant. My apologies @cal-smith! In case if you can do a favor - Can the sorting be based on directory name then file name, instead of the import name? In this way we can avoid a condition e.g. UI shell imports being split to two places.

Reply:
For sure, my bad if I messed it all up 😅 I'll do a more nuanced sort

Reply:
Not a problem @cal-smith! It's all my bad missing where the deleted portions went 🙇

Reply:
Should be sorted a lot better now :+1: